tests/posix/common/src/clock.c

```diff
@@ -38,7 +36,7 @@ ZTEST(posix_apis, test_posix_clock)
 	}
 
 	/*TESTPOINT: Check if POSIX clock API test passes*/
-	zassert_equal(secs_elapsed, (2 * SLEEP_SECONDS),
+	zassert_equal(secs_elapsed, SLEEP_SECONDS,
```

Comment:
maybe this test update should be part of the commit that changes usleep behavior? Otherwise, this test will fail for a couple commits in git history

Reply:
I might just reorder the commits. Normally we should keep test commits and code commits separate.

Reply:
Fixed!